### ISSUE TITLE
Suggestions: Donation Link

### ISSUE THREAD

**Issue by Gregoyle** [Drive4ik/simple-tab-groups#120]
This addon is absolutely great. I'm surprised you don't have a donate link on the addon page. I'd gladly donate to the cause as would dozens of others I'm sure. Is there a reason the donation information is absent?

**Comment by Drive4ik** [comment]
Hi! Thank you, I'm very pleased that you want to help with development, but it seems to me that this add-on is still young and has some bugs and it has not yet fully revealed itself. For now, my conscience does not allow me to accept payment.
But soon everything will change. I want to release a stable version with the tabs hidden and then maybe I'll make donations.
But also there is another problem - I live in Ukraine, and paypal here does not allow accepting payments. I think over the decision.
Can you have any suggestions?

**Comment by nd-** [comment]
Liberapay is a great solution for donate and receive payments, created and maintained by a real nice guy.
And it's 100% FOSS (unlike Paypal).
https://en.liberapay.com/

**Comment by mrietveld** [comment]
"For now, my conscience does not allow me to accept payment". 

I'm pretty sure that there are at least a few people who disagree with you. :D Think of it this way: from my standpoint, if I donate, then there's a strong chance that you'll have more time to work on the project. 

On the other hand, sometimes getting paid for something takes the fun (and creativity) out of it. That's a very good reason *not* to accept donations. 

One other possibility is https://www.bountysource.com/

**Comment by kparal** [comment]
@Drive4ik You should absolutely get rewarded for the awesome work you do. I still don't see any donation link. Here are some suggestions from Mozilla how to accept donations on extensions:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Distribution/Make_money_from_browser_extensions#Ask_for_donations
If Paypal is not available in your country, perhaps [Patreon](https://www.patreon.com) is? It's a very popular option nowadays.
You can also accept donations directly through Github, even though I expect only developers to use such an option (thus limiting your reach):
https://github.blog/2019-05-23-announcing-github-sponsors-a-new-way-to-contribute-to-open-source/
Another option is [Flattr](https://flattr.com), but that's another niche platform.

**Comment by esa1975** [comment]
I know this is an old topic but I agree 100% that you should accept donations. I can't imagine using a browser without this so it's an absolutely essential add-on to me. I can't thank you enough for all your effort on this. It really is an amazing piece of software. If you do get a donation link working I will definitely send you something.

**Comment by victoriastuart** [comment]
I completely agree with the other comments: this is the second time I've actually (actively) searched for a donation link for STG.

It's a wonderful Firefox extension -- perhaps my most used.  I have ~1600 tabs open, organized via STG (which after approximately a year or so of daily use has been working, flawlessly).

Honestly, I would not be able to organize those tabs without STG, which beautifully declutters Firefox -- absolutely a functional necessitity.

**Comment by grahamperrin** [comment]
https://github.com/Drive4ik/simple-tab-groups/issues/120#issuecomment-372055606

👍 to Liberapay and the nice guy. Unfortunately: 

https://github.com/Drive4ik/simple-tab-groups/issues/120#issuecomment-371823232

> … Ukraine, and paypal here does not allow accepting payments. …

The same, now, might be true for Stripe. 

https://en.liberapay.com/about/global ▶ https://stripe.com/global

**Comment by grahamperrin** [comment]
https://github.com/Drive4ik/simple-tab-groups/issues/344#issue-416524430 draws attention to https://github.com/BlitzKraft/saythanks.io where the first example (there are others) involves **Cash App**. According to https://almvest.com/blog/cash-app-available-countries-guide-%F0%9F%A5%87-all-the-countries-it-works-in/ (2020): 

> … only work in the UK and the US. You can only use it to send money within these countries …

– that's probably true, given those two countries alone in the sidebar at https://cash.app/legal/us/en-us/privacy

----

Still seeking a Ukraine-friendlly option … I'd **love** to get donations through to @Drive4ik (and/or any other person who has significantly improved the extension – glancing at https://github.com/Drive4ik/simple-tab-groups/pulls?q=is%3Apr for example).

**Comment by grahamperrin** [comment]
https://github.com/Drive4ik/simple-tab-groups/issues/120#issuecomment-393816200

### [Bountysource](https://www.bountysource.com/)

Hopeful? UA (Ukraine) at: 

https://github.com/bountysource/core/blob/f0a4c78207d40675f412fed6f45935e5dde0196d/app/assets/javascripts/common/constants/countries.js#L931

**Comment by grahamperrin** [comment]
### [IssueHunt](https://issuehunt.io/)

https://spectrum.chat/issuehunt/general/about-withdrawal~a7bb24a1-6816-47f3-8edc-bf29cd81c0d8

> … PayPal. Other payment methods are not supported.

**Comment by grahamperrin** [comment]
Bountysource gained a poor reputation. 

https://help.nextcloud.com/t/-/90403/2 steered readers to: 

* https://wiki.snowdrift.coop/market-research/other-crowdfunding (not dated, probably 2016)
* [Issuehunt](https://issuehunt.io/), which might have originated after the Snowdrift page. 

https://help.nextcloud.com/t/-/90403/4 steered readers to: 

* [Open Collective - Make your community sustainable. Collect and spend money transparently.](https://opencollective.com/)

**Comment by RokkuCode** [comment]
I would love to donate because I am a long time user of your extension. I am very glad it exists and works very well for me. The new firefox tab groups are chrome style tab groups which i don't like and would not work for my many tabs. So i will be a long time user of your extension because the firefox solution is not working for me. Are there any news on how I can donate?
